index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Esperanto parser</title>
+	<link rel="stylesheet" href="esperanto.css">
+</head>
+<body>
+
+
+<h1>Esperanto parser</h1>
+
+<h2>Source</h2>
+<textarea id="source" cols="80" rows="20">
+Alicio, jam longan tempon sidinte apud sia fratino sur la deklivo, tre enuiĝis pro senokupo. Unu, du foje ŝi prove rigardis en la libron kiun la fratino legas, sed povis vidi en ĝi nek desegnojn nek konversaciojn, kaj "por kio utilas libro," pensis ŝi, "enhavanta nek desegnojn nek konversaciojn?" Ŝi do ekpripensis—ne tre vigle ĉar la tago estis varma, kaj ŝi sentis sin tre dormema—ĉu la plezuro fari ĉenon el lekantetoj valorus la laboron sin levi kaj kolekti lekantetojn, kiam tutapude preterkuris Blanka kuniklo kun paleruĝaj okuloj.
```

Comment:
Default text from _La Aventuroj de Alicio en Mirlando_ (Alice's Adventures in Wonderland) via [libraro](http://www.libraro.net/read/lewis_carroll/la_aventuroj_de_alicio_en_mirlando/).